src/id3/v2/tag.rs

```diff
@@ -285,6 +288,30 @@ impl ID3v2Tag {
 
 		(None, None)
 	}
+
+	fn insert_frame(&mut self, item: TagItem) {
```

Comment:
The purpose of this function is to insert an _item_ by mapping it to the underlying representation, but it does not (always) insert a frame -> `insert_item()`

Reply:
I fixed this part.